compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/ir/DeclarationFactory.kt

```diff
@@ -11,7 +11,7 @@ import org.jetbrains.kotlin.ir.types.IrType
 interface DeclarationFactory {
     object FIELD_FOR_OUTER_THIS : IrDeclarationOriginImpl("FIELD_FOR_OUTER_THIS", isSynthetic = true)
 
-    fun getFieldForEnumEntry(enumEntry: IrEnumEntry, entryType: IrType): IrField
+    fun getFieldForEnumEntry(enumEntry: IrEnumEntry): IrField
```

Comment:
Is it also unused in K/N?

Reply:
Yes, K/N doesn't implement the `DeclarationFactory` interface.